src/List/ItemList.js

```diff
@@ -125,8 +125,13 @@ const Items = props => {
                       value={item.name}
                     />
                     {item.name}
-                    <button onClick={e => handleDelete(e)} value={item.name}>
-                      X{" "}
+                    <button
+                      role="button"
+                      arialabel="delete item"
```

Comment:
I don't know for sure, but this might need to be `airaLabel` instead of `arialabel`?

Reply:
Interestingly, React treats aria labels a bit differently:
https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html#data-and-aria-attributes